src/meds_torch/utils/custom_normalization.py

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env python
 """Transformations for normalizing MEDS datasets, across both categorical and continuous
-dimensions."""
+dimensions.
+
+DO NOT RUN THIS WITH PARALLELISM
```

Comment:
Might be worth putting this in the main function docstring; then it will show up on help strings too I think.